pkgs/applications/science/biology/ants/default.nix

```diff
@@ -20,6 +20,11 @@ stdenv.mkDerivation rec {
   checkPhase = "ctest";
   doCheck = false;
 
+  postFixup = ''
+    sed -e "6a \$ENV{'ANTSPATH'}=\"$out/bin/\";" -i $out/bin/antsNeuroimagingBattery
+    grep -l ANTSPATH $out/bin/*sh | xargs sed -e "2a export ANTSPATH=$out/bin/" -i --
```

Comment:
the following is probably more robust:

```
for file in $out/bin/*; do
  wrapProgram $file --set ANTSPATH  "$out/bin/"
done
```
it could be probably also used for `sed -e "6a \$ENV{'ANTSPATH'}=\"$out/bin/\";" -i $out/bin/antsNeuroimagingBattery`